model/VocabularyConfig.php

```diff
@@ -31,41 +31,97 @@ class VocabularyConfig extends BaseConfig
     public function __construct($resource, $globalPlugins=array())
     {
         $this->resource = $resource;
-        $plugins = $this->resource->allLiterals('skosmos:usePlugin');
+        $this->globalPlugins = $globalPlugins;
+        $this->setParameterizedPlugins();
+        $pluginArray = $this->getPluginArray();
+        $this->pluginRegister = new PluginRegister($pluginArray);
+    }
+
+    /**
+     * Get an ordered array of plugin names with order configured in skosmos:vocabularyPlugins
+     * @return array of plugin names
+     */
+    public function getPluginArray() : array
+    {
         $pluginArray = array();
+
+        $vocabularyPlugins = $this->resource->getResource('skosmos:vocabularyPlugins');
+        if ($vocabularyPlugins) {
+            foreach ($vocabularyPlugins as $plugin) {
+                if ($plugin instanceof EasyRdf\Literal) {
+                    $pluginName = $plugin->getValue();
+                    array_push($pluginArray, $pluginName);
```

Comment:
Why not simply do

    $pluginArray[] = $plugin->getValue();

instead of using `array_push` and defining a single-use variable? Using `$array[] = ...` is what the [PHP documentation for array_push](https://www.php.net/manual/en/function.array-push.php) recommends when you're only adding a single value:

> Note: If you use array_push() to add one element to the array, it's better to use $array[] = because in that way there is no overhead of calling a function.